### PR TITLE
LC-263 CSVConnector error handling

### PR DESCRIPTION
**Pull request by h0rban**

### CODE REVIEW

File: lucille-core/src/main/java/com/kmwllc/lucille/connector/CSVConnector.java [hunk 1]
```diff
@@ -114,7 +117,11 @@ public void execute(Publisher publisher) throws ConnectorException {
   }
 
   private void processFile(String filePath, Publisher publisher) throws ConnectorException {
-    int lineNum = 0;
+
+    if(lineNum != 0) {
```
Comment (rseitz) [inline lucille-core/src/main/java/com/kmwllc/lucille/connector/CSVConnector.java:121]: please add a space after the "if"

**Review by rseitz — APPROVED** [review]
Approving but would like to kyle to review as well before we merge

**Review by kyle-kmw — APPROVED** [review]

**Review by rseitz — CHANGES_REQUESTED** [review]
After an initial parsing error, the reported line numbers of further errors are not always accurate.

If the test csv is extended like this:

a,b,c
1,2,3
4,5
6,7,8
9,10,11
"{}",12,"
",13,14
",hi,hello
a,b,c"
15,16,17
18,19,20
21,22
23,24
25,26,27

The output is:

23/11/01 09:32:22 WARN CSVConnector: Line 3 of the csv has a different number of columns than columns in the header.
23/11/01 09:32:22 WARN CSVConnector: Line 6 of the csv has a different number of columns than columns in the header.
23/11/01 09:32:22 WARN CSVConnector: Line 7 of the csv has a different number of columns than columns in the header.
23/11/01 09:32:22 WARN CSVConnector: Line 10 of the csv has a different number of columns than columns in the header.
23/11/01 09:32:22 WARN CSVConnector: Line 11 of the csv has a different number of columns than columns in the header.

**Comment by rseitz** [comment]
Closing this PR in favor of https://github.com/kmwtechnology/lucille/pull/63

**Pull request closed without merging**